src/nvim/ops.c

```diff
@@ -3644,6 +3644,10 @@ int do_join(size_t count,
    */
   for (t = (linenr_T)count - 1;; t--) {
     cend -= currsize;
+	if(cend == NULL || curr == NULL){
```

Comment:
Ok. Thank you for the help. I was already in master branch on my local computer because I thought I needed to merge the files first. Thank you for the clarification. I just pushed a new update, hopefully you can see it.